src/app/course/components/editor/editor.component.ts

```diff
@@ -15,6 +15,8 @@ export enum CourseEditorMode {
 export class CourseEditorComponent implements OnInit {
 	@Input() public mode: CourseEditorMode;
 	@Input() public courseId?: number;
+	@Input() public doAfterSave: (Course) => Promise<void> = Promise.resolve;
```

Comment:
Check HW requirements for Editor form, please

Reply:
![image](https://user-images.githubusercontent.com/81073/71117858-01876080-21e8-11ea-8a92-5821140865cc.png)